src/meta/src/controller/mod.rs

```diff
@@ -64,14 +64,26 @@ pub struct SqlMetaStore {
     pub endpoint: String,
 }
 
-pub const IN_MEMORY_STORE: &str = "sqlite::memory:";
-
 impl SqlMetaStore {
     /// Connect to the SQL meta store based on the given configuration.
     pub async fn connect(backend: MetaStoreBackend) -> Result<Self, sea_orm::DbErr> {
         Ok(match backend {
             MetaStoreBackend::Mem => {
-                let conn = sea_orm::Database::connect(IN_MEMORY_STORE).await?;
+                const IN_MEMORY_STORE: &str = "sqlite::memory:";
+
+                let mut options = sea_orm::ConnectOptions::new(IN_MEMORY_STORE);
```

Comment:
A more elegant solution might be "Temporary Databases"? No additional options are needed.

https://www.sqlite.org/inmemorydb.html

Reply:
> Temporary databases are automatically deleted when the connection that created them closes.

Seems that this still holds.

> No additional options are needed.

Actually those options are for the connection pool by `sqlx`, not SQLite.